### DISCUSSION TITLE
how find text added in commit

### DISCUSSION THREAD

**Discussion by jindamvani** [community/community#55077] [New to GitHub]
I am unable to find text added on https://github.com/plateaukao/einkbro/commit/2c5b216d371a702d8d2475e5545a7fb9a4f0b88a Where can i find?
For example:Feature,  Reader mode font configuration (not stable yet)

**Comment by airtower-luna** [comment]
The diff for that commit shows up with the link you posted.

If you mean a release log for the new version, check the release or tag description for that version.

**Reply by jindamvani** [reply]
@airtower-luna 
Can you please show how to check the release or tag description for that version?

**Reply by airtower-luna** [reply]
Go to the main page of the repository, and look at the "Releases" section in the sidebar:

![grafik](https://github.com/community/community/assets/5814919/7f2b0b38-da12-4660-8db0-b271ca46655d)

**Reply by jindamvani** [reply]
@airtower-luna yes, i clicked it. But commit https://github.com/plateaukao/einkbro/commit/2c5b216d371a702d8d2475e5545a7fb9a4f0b88a shows only "Showing 1 changed file with 2 additions and 2 deletions. " 
* https://github.com/plateaukao/einkbro/releases/tag/v10.5.0 text feature & fix are not shown in commit

**Reply by airtower-luna** [reply]
That is correct. The "changes in that one commit" are a very different thing than "changes since the last release".

If you want to see the latter you need to compare the repository at those tags, not look at a single commit. The easiest way is to use the "Compare" menu on the release page (top right).

Locally you'd do the same thing with `git diff OLD_TAG..NEW_TAG` (substitute the actual tags).
